model/database/registry.yaml

```diff
@@ -519,7 +519,8 @@ groups:
               value: "upsert"
               stability: experimental
         stability: experimental
-        brief: Cosmos DB Operation Type.
+        deprecated: "Removed."
```

Comment:
deprecated attributes should be moved to https://github.com/open-telemetry/semantic-conventions/blob/main/model/database/deprecated/registry-deprecated.yaml